benchexec/tools/lazycseq.py

```diff
@@ -24,6 +24,7 @@ class Tool(cseq.CSeqTool):
         "pycparser",
         "pycparserext",
         "modules",
+        "backends"
```

Comment:
Our style requires a trailing comma:
```suggestion
        "backends",
```